diff --git a/Makefile b/Makefile
index c4a9c20..f8bff88 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,10 @@
 # Keep this first line.
 GOSSAMER_PATH=gossamer
 
+ifdef MOVEMENT_CONFIG
+CFLAGS += -DMOVEMENT_CONFIG='"$(MOVEMENT_CONFIG)"'
+endif
+
 # Which board are we building for? Commented out to force a choice when building.
 # Options are:
 # - sensorwatch_pro
diff --git a/movement.c b/movement.c
index cafa09f..9758b93 100644
--- a/movement.c
+++ b/movement.c
@@ -45,7 +45,11 @@
 #include "delay.h"
 #include "thermistor_driver.h"
 
-#include "movement_config.h"
+#ifndef MOVEMENT_CONFIG
+#define MOVEMENT_CONFIG "movement_config.h"
+#endif
+
+#include MOVEMENT_CONFIG
 
 #include "movement_custom_signal_tunes.h"
 
